aws/lambda/log-classifier/src/rule_match.rs

```diff
@@ -39,9 +39,11 @@ impl SerializedMatch {
                 break;
             }
 
-            let l = log.lines.get(&i).unwrap();
-            if l.starts_with("+") {
-                context.push(l.clone());
+            if !log.lines.get(&i).is_none() {
```

Comment:
It's been a while since I've used rust, but perhaps this could be turned into 
```
if let Some(l) = log.lines.get(&i) {
  ...
}
```

Reply:
Let me try, I'm a rust noob, so I have a feeling that I code it like I do Python instead of like Rust lol